pkgs/by-name/ax/ax25ms/package.nix

```
@@ -0,0 +1,60 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, autoconf
+, protobuf
+, pkg-config
+, grpc
+, libtool
+, which
+, automake
+, libax25
+}:
+
+stdenv.mkDerivation {
+  pname = "ax25ms";
+  version = "unstable-2024-04-28";
```

❗ Unstable versions should be prefixed with the last available version or `0` if no version is available from the package.
```suggestion
  version = "0-unstable-2024-04-28";
```